samples/v1/types_config.go

```diff
@@ -138,7 +138,10 @@ const (
 	// of this operator, it is equivalent to X86Architecture, which is kept for historical/migration
 	// purposes
 	AMDArchitecture = "amd64"
-	// PPCArchitecture is the value used to specify the x86_64 hardware architecture
+	// ARMArchitecture is the value used to specify the aarch64 hardware architecture
+	// in the Architectures array field.
+	ARMArchitecture = "arm64"
```

Comment:
@gabemontero correct me if I'm wrong, but I believe we want the values to align with golang arch values whenever possible.

Reply:
arm64 is the golang name for aarch64.

Reply:
@adambkaplan we do want it to align with `runtime.GOARCH` where `runtime` is the golang package we import 

Based on @yselkowitz 's comment I'm inferring he has confirmed this.